front/app/modules/commercial/insights/admin/containers/Insights/Details/index.tsx

```diff
@@ -57,6 +57,8 @@ const DetailsInsightsView = ({
 
   const category = query.category;
   const search = query.search;
+  const keywords: string[] =
```

Comment:
For some reason if there is only one item in the array qs stores it as a simple string so I have to check for that here.